docs/extend/index.md

```diff
@@ -163,4 +163,152 @@ in subdirectory `rootfs`.
 
 After that the plugin `<plugin-name>` will show up in `docker plugin ls`.
 Plugins can be pushed to remote registries with
-`docker plugin push <plugin-name>`.
\ No newline at end of file
+`docker plugin push <plugin-name>`.
+
+#### Example creating a Rexray volume plugin for Amazon EBS
```

Comment:
I think a EBS volume plugin deserves its own page. Can you please add a new file under `docs/extend/EBS_volume.md` and move this content? It also helps avoid confusion in this page, which uses `sshfs` to demonstrate install and create.